src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/model/commonplatform/Address.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.probation.courtcasematcher.messaging.model.commonplatform;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+public class Address {
+    private final String address1;
```

Comment:
Assuming that validation is being done on one of the other tickets because address1 is required. Won'r repeat this comment